lib/exsom/xsd.ex

```diff
@@ -8,11 +8,4 @@ defmodule Exsom.XSD do
   """
   @spec compile(xsd :: Exsom.xsd, []) :: {atom, Exsom.model}
   def compile(xsd, opts \\ []), do: :erlsom.compile_xsd(xsd, opts)
-
-  @doc ~S"""
-  """
-  @spec find(atom, atom, atom, atom) :: any
-  def find(namespace, location, include_dirs, include_list) do
```

Comment:
Hmm, I don't think so, there is no `:erlsom.find` function. I'm still planning to look at the list of required features you gave me, and then add stuff based on that.